src/main/java/world/bentobox/bentobox/listeners/PortalTeleportationListener.java

```diff
@@ -39,15 +41,18 @@ public PortalTeleportationListener(@NonNull BentoBox plugin) {
 
     /**
      * Handles non-player portal use.
-     * Currently disables portal use by entities to prevent dupe glitching.
+     * Dropped items cannot teleport
      *
      * @param e - event
      */
     @EventHandler(priority = EventPriority.LOW, ignoreCancelled = true)
     public void onEntityPortal(EntityPortalEvent e) {
-        if (plugin.getIWM().inWorld(e.getFrom())) {
-            // Disable entity portal transfer due to dupe glitching
-            e.setCancelled(true);
+        plugin.logDebug(e.getEventName());
+        if (plugin.getIWM().inWorld(e.getFrom()) && !e.getEntityType().equals(EntityType.DROPPED_ITEM)) {
```

Comment:
I would like to be able to send dropped items too.

Reply:
The reason I kept that one out was because I suspect dupe glitching still works. i.e., you throw a diamond though the portal and you get one on one side and one on the other.

Reply:
I think that preventing dupes does not fall in the scope of BentoBox. If we remove that safety, admins will able to rely on other plugins or on Spigot/Paper to fix the dupe. Because the current implementation is mostly about preventing dupes from happening whatsoever, regardless of whether it is already fixed or not.